homeassistant/components/overkiz/quality_scale.yaml

```diff
@@ -0,0 +1,68 @@
+rules:
+  # Bronze
+  config-flow: done
+  brands: done
+  dependency-transparency: done
+  common-modules: done
+  has-entity-name: done
```

Comment:
I would recommend moving the `_attr_name = None` to the classes which actually have the device name as entity name, this way you can easily add translations